versions/1/data_stream/fields/fields.spec.yml

```
@@ -77,22 +93,178 @@ spec:
+          into a group field, which can then be queried as a single field.
+        type: string
+
+      enabled:
+        description: >
+          The enabled setting, which can be applied only to the top-level
+          mapping definition and to object fields, causes Elasticsearch to skip
+          parsing of the contents of the field entirely. The JSON can still be
+          retrieved from the _source field, but it is not searchable or stored
+          in any other way.
+        type: boolean
+
+      dynamic:
+        description: >
+          The dynamic parameter controls whether new fields are added
+          dynamically. It accepts the following values:
+
+          true -  New fields are added to the mapping (default).
+
+          runtime - New fields are added to the mapping as runtime fields. These
+          fields are not indexed, and are loaded from _source at query time.
+
+          false - New fields are ignored. These fields will not be indexed or
+          searchable, but will still appear in the _source field of returned
+          hits. These fields will not be added to the mapping, and new fields
+          must be added explicitly.
+
+          strict -  If new fields are detected, an exception is thrown and the
+          document is rejected. New fields must be explicitly added to the
+          mapping.
+        default: true
+        enum:
+          - true
+          - false
+          - strict
+          - runtime  # Not supported at this time by Fleet.
+
+      scaling_factor:
+        description: >
+          The scaling factor to use when encoding values. Values will be
+          multiplied by this factor at index time and rounded to the closest
+          long value. For instance, a scaled_float with a scaling_factor of 10
+          would internally store 2.34 as 23 and all search-time operations
+          (queries, aggregations, sorting) will behave as if the document had a
+          value of 2.3. High values of scaling_factor improve accuracy but also
+          increase space requirements. Only valid for 'type: scaled_float'.
+        type: integer
+        default: 1000
+
+      analyzer:
+        description: >
+          Name of the analyzer to use for indexing. Unless search_analyzer is
+          specified this analyzer is used for both indexing and searching.
+          Only valid for 'type: text'.
+        type: string
+
+      search_analyzer:
+        description: >
+          Name of the analyzer to use for searching. Only valid for 'type: text'.
+        type: string
+
+      multi_fields:
+        description: >
+          It is often useful to index the same field in different ways for
+          different purposes. This is the purpose of multi-fields. For instance,
+          a string field could be mapped as a text field for full-text search,
+          and as a keyword field for sorting or aggregations.
+
+          Only honored for 'type: text/keyword/wildcard'.
```

After https://github.com/elastic/kibana/pull/128391 `match_only_text` will be also supported.
```suggestion
          Only honored for 'type: text/keyword/wildcard/match_only_text'.
```

Or do you mean that ES only supports multi fields on these types? Then maybe we have to reconsider the latest changes about this in the spec and fleet.

---

This is documenting Fleet. For clarity I'll change the wording to "Fleet honors this for `keyword`, `match_only_text', `text`, and `wildcard` types."